package.json

```diff
@@ -17,17 +17,21 @@
     "core-js": "2",
     "es6-math": "^1.0.0",
     "history": "^4.7.2",
-    "otp-react-redux": "^0.13.7",
+    "leaflet": "^1.6.0",
+    "leaflet.polylinemeasure": "github:ppete2/Leaflet.PolylineMeasure",
```

Comment:
Let's not add leaflet polylinemeasure here. Instead let's remove the css import in main.js (?).

Reply:
Updated in commit 2295153.